### PR TITLE
fix php8 warning

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

**Comment by frederic34** [comment]
when calling user->getNomUrl() need to have statut defined

File: htdocs/core/ajax/ajaxdirtree.php [hunk 1]
```diff
@@ -288,6 +288,7 @@
 			print '<td class="center">';
 			$userstatic->id = $val['fk_user_c'];
 			$userstatic->lastname = $val['login_c'];
+			$userstatic->lastname = $val['statut_c'];
```
Comment (hregis) [inline htdocs/core/ajax/ajaxdirtree.php:291]: @frederic34    $userstatic->statut

Comment (frederic34) [reply htdocs/core/ajax/ajaxdirtree.php]: @hregis thank's

Comment (frederic34) [reply htdocs/core/ajax/ajaxdirtree.php]: comments are always welcome

**Pull request merged**